pkg/metrics/metrics.go

```diff
@@ -81,7 +81,7 @@ func provideMetrics() {
 	http.Handle("/metrics", promhttp.Handler())
 	err := http.ListenAndServe(":9090", nil)
 	if err != nil {
-		log.Fatalf("Failed to start metrics server for pipeline run controller:%v", err)
+		klog.Fatalf("Failed to start metrics server for pipeline run controller:%v", err)
```

Comment:
We should not use the "fatal" methods. Rather log an error and perform the exit explicitly.

Reply:
Why, I thought fatal is the same as error + exit?